build/scripts/check-toolset-insertion.ps1

```diff
@@ -21,6 +21,11 @@ try
         "Microsoft.Build.Utilities.Core.dll",
         "Microsoft.DiaSymReader.Native.arm.dll"
     )
+
+    $exludedFromVsix = @(    
+        "Microsoft.DiaSymReader.Native.x86.dll", # installed by msbuild setup component
+        "Microsoft.DiaSymReader.Native.amd64.dll"  # installed by msbuild setup component
```

Comment:
Does this need to be pulled from our NuGet package as well?  The NuGet package we ship is meant to exactly mirror our VSIX installation in content.  It keeps our deployment story very simple.  If one deviates it would seem that both should.